docs/faq.md

```diff
@@ -159,6 +159,36 @@ Both Semgrep and CodeQL use static analysis to find bugs, but there are a few di
 
 See [the Semgrep development philosophy](/contributing/semgrep-philosophy/) for more about what makes Semgrep different.
 
+### Comparing Semgrep to Endor Labs
+
+* **Prioritization**: Both Endor Labs and Semgrep support the prioritization of findings so that AppSec teams focus on the most impactful findings. While both companies offer findings filters based on criteria like reachability and EPSS scores, Semgrep offers support for statuses in addition to the basic reachability statuses of **reachable** and **not reachable**, such as **always reachable** and **conditionally reachable**.
+
+  Furthermore, Semgrep Assistant uses AI to help organization admins receive information on top backlog tasks, allowing them to prioritize findings from all products, including the SAST and SCA products, not just those resulting from dependency vulnerability scans.
+
+* **Reachability for transitive dependencies**: Reachability has been a fundamental part of Semgrep Supply Chain from the beginning. Supply Chain offers advanced reachability analysis for direct dependencies in the form of dataflow reachability, offering accuracy beyond that offered by Endor Labs. This coverage is offered for seven languages and counting.
+
+  Semgrep is also developing a dependency graph that shows the relationship between dependencies, which offers additional insight into the relationships between direct dependencies, transitive dependencies, and their associated vulnerabilities.
```

Comment:
I'm not sure we can state this publicly in the docs - we avoid roadmap stuff where we can.